auth/auth-router.js

```diff
@@ -65,7 +72,13 @@ router.post("/login", (req, res) => {
           token
         });
       } else {
-        res.status(401).json({ message: "Invalid Credentials" });
+        if (!user) {
+          errors.username = "That username does not exist";
+        }
+        if (user && !bcrypt.compareSync(password, user.password)) {
+          errors.password = "Incorrect password";
+        }
+        return res.status(400).json(errors);
```

Comment:
Generally speaking, you don't tell a user WHY their authentication failed. This is a security feature to help keep brute force attacks from working.

Reply:
Changed it.